src/modules/getScores.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import GameUrl from './consts';
+import displayScores from './displayScores';
+
+const getAllScores = async () => {
+  try {
+    const response = await axios.get(GameUrl);
+    const respObj = response.data;
+    console.log(respObj.result);
```

Comment:
- [ ] Also, please avoid committing `console.log` calls. According to the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md):

![image](https://user-images.githubusercontent.com/77069011/217667637-38db4893-b2c9-4358-88fb-c4449999a4ab.png)

consider removing it 🤓 🔧